skunkworks_crow/src/main/res/values/strings.xml

```diff
@@ -131,6 +131,17 @@
     <string name="odk_destination_dir_error">The destination should not be empty</string>
     <string name="location_permission_needed">Location Permission Needed</string>
     <string name="location_settings_dialog">Enable location from the settings.</string>
+    <string name="title_reset_settings">Reset Application</string>
+    <string name="reset_database"> Reset Database</string>
```

Comment:
I don't think this will give the user many contexts about what's present in the database.
It could be like `Saved Forms( forms sent for review, forms received for review)`. I think this is pretty much we store in our table.